pkgs/development/python-modules/openstep-parser/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "openstep-parser";
+  version = "2.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "kronenthaler";
+    repo = "openstep-parser";
+    rev = "refs/tags/${version}";
+    hash = "sha256-gvfzBLLaal0Vad3C4m4wIKwJpmlhewsK4A5yeN8l6qU=";
+  };
+
+  build-system = [ setuptools ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "openstep_parser" ];
+
+  meta = with lib; {
+    description = "OpenStep plist reader/writer for Python";
+    homepage = "https://github.com/kronenthaler/openstep-parser";
+    changelog = "https://github.com/kronenthaler/openstep-parser/releases/tag/${version}";
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ ilaumjd ];
+  };
```

```suggestion
  meta = {
    description = "OpenStep plist reader/writer for Python";
    homepage = "https://github.com/kronenthaler/openstep-parser";
    license = lib.licenses.bsd2;
    maintainers = with lib.maintainers; [ ilaumjd ];
  };
```
`meta = with lib; ...` is an anti-pattern, as is including the version in the changelog. However, openstep-parser doesn't actually have any releases, so `https://github.com/kronenthaler/openstep-parser/releases` would be meaningless. I recommend you omit the changelog field.